nikola/plugins/command/import_wordpress.py

```diff
@@ -131,6 +138,14 @@ def _execute(self, options={}, args=[]):
         self.exclude_drafts = options.get('exclude_drafts', False)
         self.no_downloads = options.get('no_downloads', False)
 
+        self.auth = None
+        if options.get('download_auth', None) is not None:
+            username_password = options.get('download_auth')
+            self.auth = tuple(username_password.split(':', 1))
```

Comment:
Both the requests documentation and source code explicitly mention tuple, but not list. So I'd prefer to stick to a tuple here.